packages/dds/map/src/interfaces.ts

```diff
@@ -344,20 +345,21 @@ export interface ISharedMapEvents extends ISharedObjectEvents {
 
 /**
  * Shared map interface
+ * TODO: what are the semantics of this?
```

Comment:
I think on `SharedMap` we just do `{@inheritDoc ISharedMap.get}`. We should probably explicitly call out storing other shared objects within SharedMap through their handles only (otherwise dynamic shared objects are not live and shared). Initially (until shared tree is ready), it may be a common pattern used for composition.